src/index.js

```diff
@@ -1,8 +1,10 @@
 import * as components from './components';
+import GirderVue from './GirderVue';
 import RestClient from './rest';
 import * as utils from './utils';
 
 export {
+  GirderVue,
```

Comment:
If we import this and export it as an `install` function, people could install the plugin just by:

```javascript
import girder from '@girder/components';
Vue.use(girder);
```

Is there any reason not to do it that way?

Reply:
No.  I was going to say that is the way it is usually done.  I didn't even know you could pass the install function until I looked at the source.